script/docs/docs.ts

```diff
@@ -0,0 +1,23 @@
+
+// * If sitemap doesn't gave all the result can scrape directly from the website
+// console.log("on-process: 'getting all categories' . . .")
+// const categoryHrefs = await GetLinkCategory(BASE_URL);
+// if (categoryHrefs.length === 0){
+//     console.error("Category Href link is empty array")
+//     return
+// }
+// console.log("done: 'getting all categories' working-on: 'list all projects in each category' . . .")
+// const projects = await ListAllProjectInCategory(categoryHrefs)
+// console.log("done: 'list all projects in each category' working-on: 'get each project detail' . . .")
+import { ListAllFileInDocs } from "./list-all-file-github"
+import { SaveFileMdxGithub } from "./save-file-content"
+
+export async function StartDoc() {
+    console.log("Docs is starting . . .")
+
+    
+    let docs = await ListAllFileInDocs()
+    await SaveFileMdxGithub(docs)
```

Comment:
Try catch would be good here as coderabbitai said (better error handling), so we can directly migrate this code to Worker V2